src/topp/MapAlignerTreeGuided.cpp

```diff
@@ -0,0 +1,581 @@
+
+    All map alignment tools (MapAligner...) collect retention time data from the input files and - by fitting a model to this data - compute transformations that map all runs to a common retention time scale. They can apply the transformations right away and return output files with aligned time scales (parameter @p out), and/or return descriptions of the transformations in trafoXML format (parameter @p trafo_out). Transformations stored as trafoXML can be applied to arbitrary files with the @ref TOPP_MapRTTransformer tool.
+
+    The map alignment tools differ in how they obtain retention time data for the modeling of transformations, and consequently what types of data they can be applied to. The alignment algorithm implemented here is based on peptide identifications and applicable to annotated featureXML files. It finds peptide sequences that different input files have in common and uses them as points of correspondence between the inputs. For more details and algorithm-specific parameters (set in the INI file) see "Detailed Description" in the @ref OpenMS::MapAlignmentAlgorithmIdentification "algorithm documentation".
+
+    @see @ref TOPP_MapAlignerIdentification @ref TOPP_MapAlignerPoseClustering @ref TOPP_MapAlignerSpectrum @ref TOPP_MapRTTransformer
+
+		Note that alignment is based on the sequence including modifications, thus an exact match is required. I.e., a peptide with oxidised methionine will not be matched to its unmodified version. This behavior is generally desired since (some) modifications can cause retention time shifts.
+
+    Also note that convex hulls are removed for alignment and are therefore missing in the output files.
+
+    Since %OpenMS 1.8, the extraction of data for the alignment has been separate from the modeling of RT transformations based on that data. It is now possible to use different models independently of the chosen algorithm. This algorithm has been tested with the "b_spline" model. The different available models are:
+    - @ref OpenMS::TransformationModelLinear "linear": Linear model.
+    - @ref OpenMS::TransformationModelBSpline "b_spline": Smoothing spline (non-linear).
+    - @ref OpenMS::TransformationModelLowess "lowess": Local regression (non-linear).
+    - @ref OpenMS::TransformationModelInterpolated "interpolated": Different types of interpolation.
+
+    <B>The command line parameters of this tool are:</B> @n
+    @verbinclude TOPP_MapAlignerTreeGuided.cli
+    <B>INI file documentation of this tool:</B>
+    @htmlinclude TOPP_MapAlignerTreeGuided.html
+*/
+
+// We do not want this class to show up in the docu:
+/// @cond TOPPCLASSES
+
+class TOPPMapAlignerTreeGuided :
```

Comment:
There seems to be a mixed usage of `MapAlignerTreeGuided` and `MapAlignerTree` throughout.
Please pick one of them and stick to it for internal class names, filenames, INI files, documentation etc...